fearless/Common/Network/Subquery/SubqueryHistoryOperationFactory.swift

```diff
@@ -0,0 +1,145 @@
+import Foundation
+import RobinHood
+import FearlessUtils
+
+final class SubqueryHistoryOperationFactory {
+    let url: URL
+    let filter: WalletHistoryFilter
+
+    init(url: URL, filter: WalletHistoryFilter) {
+        self.url = url
+        self.filter = filter
+    }
+
+    private func prepareExtrinsicInclusionFilter() -> String {
+        """
+        {
+          or: [
+            {
+                  extrinsic: {isNull: true}
+            },
+            {
+              not: {
+                and: [
+                    {
+                      extrinsic: { contains: {module: "balances"} } ,
+                        or: [
+                         { extrinsic: {contains: {call: "transfer"} } },
+                         { extrinsic: {contains: {call: "transferKeepAlive"} } },
+                         { extrinsic: {contains: {call: "forceTransfer"} } },
+                      ]
+                    }
+                ]
+               }
+            }
+          ]
+        }
```

Comment:
Can we load it from a file maybe?

Reply:
It would create complexity because this is only a part of whole request which can have parameters to replace. I think we could consider a library to deal with Graph QL in future.